public/app/features/alerting/unified/components/rule-editor/CustomAnnotationField.tsx

```diff
@@ -0,0 +1,39 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Input, useStyles2 } from '@grafana/ui';
+
+interface CustomAnnotationFieldProps {
+  field: { onChange: () => void; onBlur: () => void; value: string; name: string };
+}
+
+const CustomAnnotationField = ({ field }: CustomAnnotationFieldProps) => {
```

Comment:
I would change the name to something like `CustomAnnotationFieldHeader` or something like that as we are not rendering only the header.